source/elens.f90

```diff
@@ -558,34 +558,37 @@ subroutine parseRadialProfile(ifile)
     write(lerr,"(a,i0)") "ELENS> ERROR radius not in increasing order at ii=",ii
     goto 30
   end if
-  if(tmpJ>=0.0) then
-    ii=ii+1
-    if(ii>elens_radial_dim) then
-      iErr = 2
-      write(lerr,"(a,i0,a,i0)") "ELENS> ERROR too many points in radial profile: ",ii, &
-           ". Max is ",elens_radial_dim
-      goto 30
-    end if
-    elens_radial_profile_nPoints(ifile) = ii
-    elens_radial_profile_R(ii,ifile) = tmpR
-    elens_radial_profile_J(ii,ifile) = tmpJ
+  ii=ii+1
+  if(ii>elens_radial_dim) then
+    iErr = 2
+    write(lerr,"(a,i0,a,i0)") "ELENS> ERROR too many points in radial profile: ",ii, &
+         ". Max is ",elens_radial_dim
+    goto 30
   end if
+  elens_radial_profile_nPoints(ifile) = ii
+  elens_radial_profile_R(ii,ifile) = tmpR
+  elens_radial_profile_J(ii,ifile) = tmpJ/c1e2 ! from [A/cm2] to [A/mm2]
 
   goto 10
 
 20 continue
 
   call f_freeUnit(fUnit)
-  write(lout,"(a,i0,a)") "ELENS> ...acquired ",elens_radial_profile_nPoints(ifile),"points."
+  if ( elens_radial_profile_nPoints(ifile).eq.0 ) then
```

Comment:
This is just nitpicky, but could we at least avoid using fortran 77 and older logical operations like `.eq.` in new code?

Reply:
promised.
Could you merge this? I cannot